src/worldcereal/openeo/postprocess.py

```diff
@@ -64,6 +198,12 @@ def smooth_probabilities(
             )
             probabilities_vals[class_idx][excluded_mask] = 0
 
+        # Sum of probabilities should be 1
+        probabilities_vals = probabilities_vals / probabilities_vals.sum(axis=0)
+
+        # Cast back to uint16
+        probabilities_vals = np.round(probabilities_vals * 100.0).astype("uint16")
```

Comment:
can we do this in one go?
something like
np.round(`probabilities_vals / probabilities_vals.sum(axis=0) * 100.0).astype(np.uint16)`?